wcomponents-test-lib/src/main/java/com/github/bordertech/wcomponents/test/selenium/by/ByWTabSet.java

```diff
@@ -0,0 +1,24 @@
+package com.github.bordertech.wcomponents.test.selenium.by;
```

Comment:
I am left a little perplexed by this By extension. Do you want to find a WTabSet or do you want to find another element **by** using a tabset. The use of `By` would indicate the desire to find another element by using a WTabSet as the means to search, which I don't think makes sense. 

Did you mean to add a WTabSet extension of `WebElement`? That would make more sense.